### PR TITLE
First round of code tests.

### PR DESCRIPTION
**Pull request by davidnewhall**
Since @Fuochi asked about tests.. here we go. Feedback welcomed. 

Still going to add a few more before I'm done with this contribution.

### CODE REVIEW

File: lidarr/sonarr_test.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package lidarr_test
```
Comment (Fuochi) [inline lidarr/sonarr_test.go:1]: change filename to lidarr_test.go

Comment (davidnewhall) [reply lidarr/sonarr_test.go:1]: haha nice catch!

Comment (davidnewhall) [reply lidarr/sonarr_test.go:1]: I actually changed GetInto in another branch, and it's going to ruin all these tests. I need to merge the branches and fix this. I appreciate you taking a look, but please check back in a couple days after I get things squared away. I have a lot of pending stuff going on in this library right now.

**Comment by Fuochi** [comment]
Hi, thanks for the effort!
Generally speaking I'd say it's fine. However, what do you think of having lists of tests for each method?
This way, it's possible to leverage the same test function to run multiple tests. Something like:
```
tests := map[string]struct {
		path string
		response          apparr.CommandResponse
		withError          error
		assertFunctions []func() // or some parameter to test
	}{
        "200": {
        //define here the cases
    }
}

for tName, t := range tests {
// move here the actual test
}
```

**Comment by davidnewhall** [comment]
> However, what do you think of having lists of tests for each method?

This sounds ideal, and probably what I'll do when I add more than 1 test haha.

**Pull request merged**